src/core/managers/TransactionManager/TransactionManager.ts

```diff
@@ -55,6 +56,19 @@ export class TransactionManager {
     }
   };
 
+  public track = async (
+    signedTransactions: Transaction[],
+    options?: { enableToasts: boolean }
```

Comment:
disableToasts boolean default undefined ?

Reply:
i change it to default {enableToasts:true}